lib/database/adapters/LhcPeriodStatisticsAdapter.js

```diff
@@ -30,8 +30,9 @@ class LhcPeriodStatisticsAdapter {
      * @returns {LhcPeriodStatistics} Converted entity object.
      */
     toEntity(databaseObject) {
-        const { id, avgCenterOfMassEnergy, distinctEnergies, lhcPeriod } = databaseObject;
-        const entity = { id, avgCenterOfMassEnergy, distinctEnergies };
+        const { id, avgCenterOfMassEnergy, lhcPeriod } = databaseObject;
+        const distinctEnergies = databaseObject.get('distinctEnergies');
+        const entity = { id, avgCenterOfMassEnergy, distinctEnergies: distinctEnergies?.split(',').map((energy) => Number(energy)) ?? [] };
```

Comment:
Why don't you want to simply include runs's column you are interested in?

Reply:
/**
 * Reason for calculation of runs lhc beam distinct energies this way (using literals) is:
 * There is transitive association: `runs -> lhc_periods -> lhc_periods_statistics`.
 * Including runs in queries below causes that sequelize build internaly subquery to fetch data.
 * This subquery is an `SELECT ... FROM lhc_periods_statistics`,
 * but after adding ordering by `lhcPeriod.name` sequelize puts the `ORDER BY` clause inside that subquery,
 * what leads to error, because lhcPeriod is not in scope of the subquery.
 * Mentioned can be avoid by providing flag:  `subQuery: false`,
 * but this breaks limit clause, so pagination.
 * There is related issue for reference: https://github.com/sequelize/sequelize/issues/11983
 */